common/src/main/java/com/microsoft/identity/common/internal/util/ProcessUtil.java

```diff
@@ -55,17 +51,22 @@ private ProcessUtil() {
     public static boolean isBrokerProcess(@NonNull final Context context) {
         final String processName = getProcessName(context);
 
-        final String authAppAuthProcess = AZURE_AUTHENTICATOR_APP_PACKAGE_NAME + ":auth";
-        final String cpAuthProcess = COMPANY_PORTAL_APP_PACKAGE_NAME + ":auth";
+        final BrokerValidator brokerValidator = new BrokerValidator(context);
+        final Set<BrokerData> validBrokers = brokerValidator.getValidBrokers();
+
+        for (final BrokerData brokerData : validBrokers) {
+            final String authProcess = brokerData.packageName + ":auth";
+            return authProcess.equalsIgnoreCase(processName);
```

Comment:
This will only run for the first element in this collection...

Reply:
Addressed